api3-whitepaper.tex

```diff
@@ -612,103 +612,66 @@ \subsection{API3 DAO}
-It is also a good token distribution tool that favors token holders that participate in the system over passive ones.
-Therefore, inflationary staking rewards will be utilized to further incentivize staking.
-
-\begin{figure}[t]
-     \centering
-     \begin{subfigure}{0.9\textwidth}
-         \centering
-         \includegraphics[width=\textwidth]{fig/token/token-weekly-emission}
-         \caption{The number of tokens minted every week is exponentially decayed to reduce the annual inflation rate from $75\%$ to $2.5\%$ until the end of year 5.
-         At the end of year 5, the annual inflation rate is fixed at a constant $2.5\%$.}
-         \label{fig:token-emission}
-     \end{subfigure}
-     \begin{subfigure}{0.9\textwidth}
-         \centering
-         \includegraphics[width=\textwidth]{fig/token/token-total-supply}
-         \caption{The total supply of API3 tokens starts from 100 million and increases according to the schedule shown above in (a).}
-         \label{fig:total-supply}
-     \end{subfigure}
-    \caption{API3 token inflation schedule for staking rewards.}
-    \label{fig:token}
-\end{figure}
-
-In essence, inflationary rewards force token holders to stake to preserve the value of their tokens.
-However, staking is risky due to the funds being used as collateral, and forces the staker to participate in governance to ensure that the risk is minimized (a similar mechanic is proposed recently in~\cite{aave}).
-As a combination of the two, an inflationary governance token used as collateral forces all token holders to participate in governance, which is ideal because it maximizes the decentralization of governance.
-Furthermore, inflationary rewards are vested for a year, which results in governing parties sharing the project’s long term interests.
-
-The inflationary rewards will start at a 75\% annual rate (1.44\% weekly), and the number of tokens minted weekly will be decayed exponentially until annual inflation rate becomes 2.5\% at the end of year 5.
-From this point on, the annual inflation will stay at a rate of 2.5\% for perpetuity (see Figure~\ref{fig:token-emission}).
-The change in the total supply of API3 tokens is illustrated in Figure~\ref{fig:total-supply}.
-The proposed inflation schedule is adapted from \cite{sip-24} and will be governable.
+A robust consensus mechanism rewards participants with the value that is captured from the enabled functionality, which creates a positive feedback loop.
+For example, Ethereum enables trustless applications.
+The miners that validate the correct operation of these applications receive the entirety of the transaction fees paid by their users.
+In our case, API3 stakers are the participants of the consensus mechanism that governs the project, which implies that they should be the sole beneficiary of dAPI monetization.
```

Comment:
Wouldn't API providers contributing to the dAPI in a way also be beneficiaries? I might remove "sole".

Reply:
API3 would have dAPI operating costs in the form of API provider compensation, gas fees, etc. Then, it (indirectly) receives what is left of this, i.e. the profit. So by the sole beneficiary, I mean the sole recipient of the operating profits. Though this may not be entirely true, because a non-staker will also benefit from the deflation. I'm 50/50 on this so just let me know what you think is the best idea based on this.

Reply:
This explanation was helpful and I don't think "sole" needs changing, but "primary" could be a nice replacement. I don't think "sole recipient of operating profits", while perhaps more descriptive, is necessary.